xml/docu_styleguide.manage.xml

```diff
@@ -153,14 +268,14 @@
     Each header of a &suse; XML file includes the entity declaration file (by
     means of an entity):
    </para>
-<screen>&lt;!ENTITY % entities SYSTEM "entity-decl.ent"&gt;
+<screen>&lt;!ENTITY % entities SYSTEM "generic-entities.ent"&gt;
 %entities;</screen>
    <example xml:id="ex-entity">
-    <title>Excerpt from an <filename>entity-decl.ent</filename></title>
-<screen>&lt;!ENTITY<co xml:id="co-declaration"/> server<co xml:id="co-definition"/> "sun<co xml:id="co-value"/>"&gt;
-&lt;!ENTITY domain  "example.org"&gt;
-&lt;!ENTITY host    "&amp;server;.&amp;domain;<co xml:id="co-entity"/>"&gt;
-&lt;!ENTITY hosttag "&lt;uri xmlns='http://docbook.org/ns/docbook'&gt;&amp;host;&lt;/uri&gt;<co xml:id="co-tag"/>"&gt;</screen>
+    <title>Excerpt from <filename>product-entities.ent</filename></title>
+<screen>&lt;!ENTITY<co xml:id="co-declaration"/>product-sp<co xml:id="co-definition"/> "Code stream<co xml:id="co-value"/>"&gt;
```

Comment:
```suggestion
<screen>&lt;!ENTITY<co xml:id="co-declaration"/>product-sp<co xml:id="co-definition"/> "1<co xml:id="co-value"/>"&gt;
```

Reply:
I would use a concrete example of a service pack number here to make it clearer.